From: Chuck Short <zulcss@ubuntu.com>
Date: Fri, 18 Jan 2008 18:21:16 +0000 (-0500)
Subject: UBUNTU: SAUCE: ata: blacklist FUJITSU MHW2160BH PL
X-Git-Url: http://kernel.ubuntu.com/git?p=ubuntu%2Fubuntu-intrepid.git;a=commitdiff_plain;h=3522f638ddff561fb46c1056fcb8c678e9e649ab

UBUNTU: SAUCE: ata: blacklist FUJITSU MHW2160BH PL
OriginalAuthor: Chuck Short <zulcss@ubuntu.com>
Bug: #175834

Signed-off-by: Chuck Short <zulcss@ubuntu.com>
Signed-off-by: Tim Gardner <tim.gardner@canonical.com>
---

Index: 2008-devel/drivers/ata/libata-core.c
===================================================================
--- 2008-devel.orig/drivers/ata/libata-core.c
+++ 2008-devel/drivers/ata/libata-core.c
@@ -4450,6 +4450,7 @@ static const struct ata_blacklist_entry 
 	{ "HTS541060G9SA00",    "MB3OC60D",     ATA_HORKAGE_NONCQ, },
 	{ "HTS541080G9SA00",    "MB4OC60D",     ATA_HORKAGE_NONCQ, },
 	{ "HTS541010G9SA00",    "MBZOC60D",     ATA_HORKAGE_NONCQ, },
+	{ "FUJITSU MHW2160BH PL", "0084001E",   ATA_HORKAGE_NONCQ, },
 
 	/* devices which puke on READ_NATIVE_MAX */
 	{ "HDS724040KLSA80",	"KFAOA20N",	ATA_HORKAGE_BROKEN_HPA, },
